cluster/examples/kubernetes/ceph/common.yaml

```diff
@@ -1125,3 +1125,10 @@ roleRef:
   name: rbd-external-provisioner-runner
   apiGroup: rbac.authorization.k8s.io
 # OLM: END CSI RBD CLUSTER ROLEBINDING
+---
+# Service account for other components
```

Comment:
We will need some OLM tags around the SA, similar to other SAs in common.yaml. 
```suggestion
# OLM: BEGIN DEFAULT SERVICEACCOUNT
# Service account for other components
...
# OLM: END DEFAULT SERVICEACCOUNT
```

Then we will need to add the new tag to the generation script [here](https://github.com/rook/rook/blob/master/cluster/olm/ceph/generate-rook-csv.sh).